src/components/DescriptionContent.jsx

```diff
@@ -16,7 +16,12 @@ export const DescriptionContent = ({
         messages.length > 0 &&
         messages.map(m => {
           return (
-            <p className="col-account-description-message">
+            <p
+              className={classNames(
+                'col-account-description-message',
+                cssClassesObject['coz-error'] && 'errors'
```

Comment:
If I understand well, we are here testing the existence of a given CSS class passed in `props` to provide another class instead ? If it make sense technically, I feel that it is not logical functionnaly, I suggest instead using an explicit `hasError` prop:
```jsx
className={classNames(
  'col-account-description-message',
  hasError && 'errors'
)}
``
What do you think ?

Reply:
Yep, that's a great idea, I fix it